opencensus/stats/aggregation_data.py

```diff
@@ -64,7 +64,7 @@ def __init__(self, count_data):
         super(CountAggregationData, self).__init__(count_data)
         self._count_data = count_data
 
-    def add_sample(self, value):
+    def add_sample(self, value, timestamp, attachments):
```

Comment:
Same here.

Reply:
They are not used, but are needed to maintain the def signature. I changed these parameter to be optional.